scripts/builds/react-router.js

```diff
@@ -88,11 +80,7 @@ const webModules = [
         plugins: ['babel-plugin-dev-expression']
       }),
       replace({ 'process.env.NODE_ENV': JSON.stringify('production') }),
-      compiler({
-        compilation_level: 'SIMPLE_OPTIMIZATIONS',
-        language_in: 'ECMASCRIPT_2018',
-        language_out: 'ECMASCRIPT_2017'
```

Comment:
I thought this one was important to have in there for babel-preset-modules to work properly. Maybe @developit can shed some light here?

Reply:
By default the value from this plugin is `NO_TRANSFORM`, which means it will not attempt to transpile any code.

Since you're using Babel for transpilation, it's best to let that system handle it.

Reply:
Ooooooh, I didn't realize the default was to do nothing. Does it automatically detect `language_in` as well? e.g. can I use advanced stuff like async/await?

Reply:
Yep! No need to configure the `language_in`. 

Closure Compiler in this mode won't transpile the code - leaves this task to Babel to do before passing the code to Closure Compiler!